src/schemas/21-526EZ/schema.js

```diff
@@ -15,139 +14,118 @@ const directDepositDef = {
-      // Should these be capitalized, or should we make sure they're lower case when we send them to the api?
-      enum: ['apo', 'dpo', 'fpo']
-    },
-    militaryStateCode: {
-      type: 'string',
-      enum: ['AA', 'AE', 'AP']
-    },
-    type: {
-      type: 'string',
-      enum: ['DOMESTIC', 'MILITARY', 'INTERNATIONAL']
-    }
-  },
-  required: ['addressLine1', 'country']
-};
+  return serviceHistory;
+})(definitions);
 
-// Apparently we don't need a suffix here.
-const fullNameDef = _.omit('properties.suffix', definitions.fullName);
+// Extracted to enable easy adding of properties for forwardingAddress
+const addressDef = definitions.pciuAddress;
 
-// Not sure this particular kind of datetime definition will be used anywhere else
-// Pattern matches datetimes like 2018-03-22T17:25:19.191Z where the fractional seconds are optional
-// NOTE: This doesn't catch invalid days like February 30th
-const datetime = {
-  pattern: '(\d{4}|XXXX)-(0[1-9]|1[0-2]|XX)-(0[1-9]|[1-2][0-9]|3[0-1]|XX)T([01]\d|2[0-3]):[0-5]\d:[0-5]\d(\.\d+)?Z',
-  type: 'string'
-}
+// Strip suffix from fullName common def
+const fullNameDef = _.merge(definitions.fullName, {
+  properties: _.omit('suffix', definitions.fullName.properties)
+});
+
+/**
+ * Strip address lines from PCIU address def for use with treatment center addresses
+ * @typedef {object} definitions
+ * @property {object} pciuAddress
+ * @param {definitions} definitions from the common schema definitions file
+ * @returns {object} the treatmentCenterAddress schema object
+ */
+const treatmentCenterAddressDef = ((definitions) => {
+  const treatmentAddressProperties = _.pick(
+    ['city', 'state', 'country'],
+    definitions.pciuAddress.properties
+  );
 
+  return _.merge(definitions.pciuAddress, { properties: treatmentAddressProperties });
+})(definitions);
 
 let schema = {
   $schema: 'http://json-schema.org/draft-04/schema#',
   title: 'SUPPLEMENTAL CLAIM FOR COMPENSATION (21-526EZ)',
   type: 'object',
   definitions: {
-    directDeposit: directDepositDef,
-    datetime,
-    // dateRange: definitions.dateRange // hopefully we can use this later
-    fullName: definitions.fullName,
-    phone: {
-      type: 'object',
-      properties: {
-        areaCode: {
-          type: 'string',
-          pattern: '\d{3}'
-        },
-        phoneNumber: {
-          type: 'string',
-          pattern: '\d{7,11}'
+    address: addressDef,
+    treatmentCenterAddress: treatmentCenterAddressDef,
+    directDeposit: _.merge(definitions.bankAccount, uniqueBankFields),
+    date: definitions.date,
+    dateRange: definitions.dateRange,
+    disabilities: _.merge(disabilitiesBaseDef, {
+      minItems: 1,
+      items: {
+        properties: {
+          secondaryDisabilities: disabilitiesBaseDef
         }
       }
-    },
+    }),
+    fullName: fullNameDef,
+    // vets-api will split into separate area code & phone number fields
+    phone: Object.assign({}, definitions.phone, { maxLength: 10 }),
```

Comment:
Should the maxLength be 11? Does the minLength need to be updated as well (default is 10)?

Reply:
Actually it looks like the updated length is [7](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/5b9e12c30318acb7d8543fa93e54c210b5bdfd9a/Products/Disability%20Compensation/engineering/EVSS/ResponseTo_questions_for_EVSS_WRT_Form_526_API.md)?

Reply:
Thanks for clarifying.

Reply:
yep, looks like min / max length should be set to 7. Thanks for the link!